backend/api/Services/MissionRunService.cs

```diff
@@ -153,6 +153,19 @@ public async Task<IList<MissionRun>> ReadMissionRunQueue(string robotId)
             return nextScheduledMissionRun;
         }
 
+        public async Task<MissionRun?> ReadPendingOrOngoingOrPausedReturnToHomeMissionRuns(string robotId)
```

Comment:
```suggestion
        public async Task<MissionRun?> ReadReturnToHomeMissionRuns(string robotId, IList<MissionStatus>? filterStatuses = null)
```
Could we maybe make the function more dynamic by having the MissionStatus values we are filtering with be an argument?

Reply:
We can, we don't need it for now, but might be usefull for the future

Reply:
I just figure it would make the code more readable, since it's a bit hard to understand the function based on its name.